### DISCUSSION TITLE
beatmap search listing: unwanted map difficulties dropdown appears when hovering beside difficulties pills

### DISCUSSION THREAD

**Discussion by aplotor** [ppy/osu#29890] [Q&A]
map difficulties dropdown appears when cursor hovers BESIDE difficulties pills in the card of a map, causing extra clutter and blocking view and access to the card(s) below it, when it was not my intent to view the difficulties dropdown

see [video](https://mega.nz/file/DkJCFJzI#9T4fh6BWRwzzLPhwLZVSi_guxtoUlJu7_4fxJxYi8Ys) for example (external link bc of github 10mb filesize limit): here i'm only trying to scroll down the list of maps; not trying to view the difficulties dropdowns for any of them, but the dropdown keeps appearing and blocking the card(s) below after almost every scroll because my cursor is in the blank space next to the difficulties pills

im not sure if this behavior was designed like this or an oversight, but i think it would be better for the dropdown to only appear if the user hovers on top of the difficulties pills, and not when hovering beside them

thanks !

**Comment by bdach** [comment]
The reason it was implemented that way in client is because the website implemented this dropdown first in that way. So if this were to change, it'd have to change in both places.

**Reply by aplotor** [reply]
thanks for your response. i opened an issue there for this https://github.com/ppy/osu-web/issues/11498
